omero/sysadmins/unix/server-install-homebrew.txt

```diff
@@ -14,7 +14,7 @@ OMERO.server Mac OS X installation walk-through with Homebrew
     the downloads page or build from source.**
 
 These instructions are implemented in a series of `automated scripts
-<https://github.com/ome/omero-install/tree/master/homebrew>`_ which
+<https://github.com/ome/omero-install/tree/develop/osx>`_ which
```

Comment:
The renaming is happening in https://github.com/ome/omero-install/pull/61. Once this PR is merged, the link should be fixed.